### ISSUE TITLE
Handoff iPad Style

### ISSUE THREAD

**Issue by alxndrdaemon** [tomaszpoliszuk/AppSwitcherController#1]
handoff is not displayed at the bottom of the switcher

**Comment by alxndrdaemon** [comment]
![image](https://user-images.githubusercontent.com/66810492/90916816-a6b99080-e396-11ea-8896-14a26dc91d89.jpeg)

**Comment by tomaszpoliszuk** [comment]
We discussed this issue via email and concluded this:

- Suggested app block exists only in iPhone "Cards" style
- iPad "Grid" style switcher doesn't have this block because suggested apps are showing in dock instead
Suggested app block needs to be rewritten from scratch so solution to this problem for now is to use [DockController](https://github.com/tomaszpoliszuk/DockController) (or any other similar tweak) and to enable iPad Dock.

I'll leave this issue open until:
- I rewrite this block
- someone contribute
- I give up with rewriting of this block

**Comment by alxndrdaemon** [comment]
Really appreciate your effort, time, reply and everything all you’re doing. Don’t worry i’ll be waiting for it ☺️I’ve been using DockController and the experience is really amazing and great, even i could say that i prefer use this one instead iphone cards style only. I just want to report a little buggy that i’ve not already seen on this days about suggested app exactly. But if i see it again, i’ll give u more details. And a suggestion: I bought FloatingDockPlus13 in the past and it could make the points on the dock or the apps closer to the dock or they go lower as the dock gets smaller. I think that i have so much space between them. If you could it would amazing if not don’t worry.I mentioned floatingdockplus 13 because this one does it automatically a little better, maybe it can be a reference, I just don't like that it thinks that all applications can be in landscape mode (especially games). Or dock controller do it but when you enable this feature; “fix icon layout” - respring - dots seems to be over the dock and change “fix icon layout” once again to « smaller dock ». And it’s perfect and awesome. Just that. Once again thanks Tomas you’re so kindly, pls if you don’t understand something that I’ve written let me know, my english is not good.

**Comment by tomaszpoliszuk** [comment]
I'm not 100% sure if i understand correctly your suggestion about Dock controller - could you rephrase it?

Also you probably should edit your comment (looks like you were replying using email - if you do that - remember to remove previous emails from reply) - if you can log into GitHub and remove everything after `Kind regards. Leo Enviado desde mi iPhone `

**Comment by alxndrdaemon** [comment]
Sorry about that. 😅
Only if I could make sure there was not so much space between the dock and the latest apps it would be great.

**Comment by alxndrdaemon** [comment]
![image](https://user-images.githubusercontent.com/66810492/91355731-73666f80-e7a3-11ea-94e5-7eadac04f958.jpeg)

**Comment by alxndrdaemon** [comment]
https://streamable.com/ywwuhn

**Comment by alxndrdaemon** [comment]
The space between dock and apps is so much better if first activate «Enable fix layout» and then "Smaller Dock" try it. 🙂 

Edit.
But when respring the iPhone come back to just "Smaller Dock". It only happens if i respring the phone, could i keep these space.

(Right pic)

**Comment by tomaszpoliszuk** [comment]
OK, I get it now :)

So first thing - unfortunately most changes are not applied dynamically and you need to respring to apply changes correctly.

and now a little explanation:

Screenshot below is cut into four parts (from left to right) to visualize how those settings look:
1 - Smaller Dock with 7 icons
2 - Enabled with 7 icons
3 - Smaller Dock with 4 icons
4 - Enabled with 4 icons

![explanation](https://user-images.githubusercontent.com/7104816/91358574-58235080-e7f3-11ea-852a-d1f4f9293886.png)

(Now when I look at it I probably should turn on labels before I took those screenshots but I hope it would be more clear now).

So basically this issue with Dock Controller you have depends on feature I hope to add someday to this tweak (ability to change settings dynamically so there would be no need for using respring).

With this feature I would be able to move icons up and down depending on how high iPad style Dock is.

I should probably find a better names or add better description for now so this would be more obvious.

If something is not clear or more discussion (about this issue with Dock Controller) is required please let's continue it [here](https://github.com/tomaszpoliszuk/DockController/issues/9) from now on.

Have a nice day!

**Comment by alxndrdaemon** [comment]
Thanks for understand my poor descriptions and really appreciate you take my comments and improve your awesome tweaks. Have a nice day too. ☀️

**Comment by tomaszpoliszuk** [comment]
Everything was fine with them - I just had to make sure that i understand your intentions, that's all, thank you for kind words 🤗

**Comment by alxndrdaemon** [comment]
It has been fixed! Thanks for your effort
